flow/include/flow/serialize.h

```diff
@@ -831,14 +836,47 @@ struct PacketBuffer : SendBuffer {
 public:
 	static PacketBuffer* create(size_t size = 0) {
 		size = std::max(size, PACKET_BUFFER_MIN_SIZE - PACKET_BUFFER_OVERHEAD);
-		uint8_t* mem = new uint8_t[size + PACKET_BUFFER_OVERHEAD];
+		uint8_t* mem = nullptr;
+		if (keepalive_allocator::isActive()) [[unlikely]]
```

Comment:
Consider hiding this conditional logic in some inline function declared in FastAlloc.h

Reply:
Good point. Will do.